tools/gen_pyi.py

```diff
@@ -71,6 +71,7 @@ def gen_pyi() -> None:
         "dataframe": "torcharrow.DataFrame",
         "end_caching": "IterDataPipe",
         "unzip": "List[IterDataPipe]",
+        "random_split": "Union[IterDataPipe, List[IterDataPipe]]",
```

Comment:
Is this return type fine? It will either be a single `IterDataPipe` or a `List[...]` depending if `target` is specified.